### PR TITLE
2024-04-24 Release

### PR DESCRIPTION
**Pull request by anitacaron**

### CODE REVIEW

**Review by matentzn — COMMENTED** [review]
These are not many changes, just two individuals added? Why making a release for that?

**Comment by anitacaron** [comment]
Are you checking the whole file, not just the last commit? There are 5 new relationships, two of which you need for OBA.

File: ro-base.obo [hunk 1]
```diff
@@ -6162,6 +6108,26 @@ range: UBERON:0000061
 is_a: RO:0002178 ! supplies
 is_a: RO:0002179 ! drains
 
+[Typedef]
+id: RO:0020202
+name: has numerator
+def: "A relationship between a ratio or proportion and its dividend." [] {http://purl.org/dc/terms/contributor="https://orcid.org/0000-0002-8280-0487"}
```
Comment (matentzn) [inline ro-base.obo:6114]: strange to use contributor relation on defs. Should be xref imo.

File: ro-base.obo [hunk 2]
```diff
@@ -5924,6 +5836,40 @@ property_value: IAO:0000112 "anticoagulant-containing test tube contains measure
...
+property_value: http://purl.org/dc/terms/contributor https://orcid.org/0000-0002-2999-0103
+property_value: http://purl.org/dc/terms/contributor https://orcid.org/0000-0002-5714-991X
+property_value: IAO:0000112 "anticoagulant-containing test tube has maximum capacity 10 ml." xsd:string
+domain: OBI:0000967
+range: COB:0000121
+
+[Typedef]
+id: RO:0017008
+name: owns
+def: "A primitive relation that holds between entities x and y in which y is at x’s full disposal." []
```
Comment (matentzn) [inline ro-base.obo:5852]: Looks to me like there is a bad char in here `x’s` instead of `x's` but no worries

File: ro-base.obo [hunk 3]
```diff
@@ -6162,6 +6108,26 @@ range: UBERON:0000061
...
+synonym: "has dividend" EXACT []
+domain: PATO:0001470
+range: PATO:0000001
+is_a: BFO:0000051 ! has part
+created_by: https://orcid.org/0000-0001-8314-2140
+
+[Typedef]
+id: RO:0020203
+name: has denominator
+def: "A relationship between a ratio or proportion and its divisor." [] {http://purl.org/dc/terms/contributor="https://orcid.org/0000-0002-8280-0487"}
```
Comment (matentzn) [inline ro-base.obo:6124]: again, wrong property for defs, should be xref

**Review by matentzn — DISMISSED** [review]
Feel free to make a release but I made some minor observations that could be fixed imo

**Comment by anitacaron** [comment]
Ok, I'll make the changes in another PR.

File: ro-base.obo (hunk 3 above)
Comment (anitacaron) [reply ro-base.obo]: done

File: ro-base.obo (hunk 2 above)
Comment (anitacaron) [reply ro-base.obo]: done

File: ro-base.obo [hunk 4]
```diff
@@ -370,8 +253,9 @@ property_value: IAO:0000112 "this investigation has participant this investigato
 property_value: IAO:0000112 "this process has participant this input material (or this output material)" xsd:string
 property_value: IAO:0000116 "Has_participant is a primitive instance-level relation between a process, a continuant, and a time at which the continuant participates in some way in the process. The relation obtains, for example, when this particular process of oxygen exchange across this particular alveolar membrane has_participant this particular sample of hemoglobin at this particular time." xsd:string
 property_value: IAO:0000118 "has_participant" xsd:string
-domain: BFO:0000003 ! occurrent
-range: BFO:0000002 ! continuant
+domain: BFO:0000003
+domain: BFO:0000015
```
Comment (anitacaron) [inline ro-base.obo:257]: is it right?

File: ro-base.obo [hunk 5]
```diff
@@ -304,8 +186,9 @@ property_value: IAO:0000118 "unfolds in" xsd:string
 property_value: IAO:0000118 "unfolds_in" xsd:string
 property_value: isDefinedBy http://purl.obolibrary.org/obo/bfo.owl
 property_value: seeAlso "https://wiki.geneontology.org/Occurs_in" xsd:anyURI
-domain: BFO:0000003 ! occurrent
-range: BFO:0000004 ! independent continuant
+domain: BFO:0000003
+domain: BFO:0000015
```
[the remark this answers is not in the document]
Comment (balhoff) [reply ro-base.obo:190]: Yes, but the result is that the actual domain is the intersection of the two. In this case BFO:0000015 is a subclass of BFO:0000003, so the actual domain is just BFO:0000015.

[the remark this answers is not in the document]
Comment (anitacaron) [reply ro-base.obo:190]: Ok, I'll fix that in another PR.

**Review by matentzn — APPROVED** [review]

**Review by cmungall — APPROVED** [review]

**Review by balhoff — APPROVED** [review]

**Pull request merged**